tests/fixtures/fixture_modflow.py

```diff
@@ -65,6 +65,15 @@ def make_mf6_simulation(gwf_model: mf6.GroundwaterFlowModel) -> mf6.Modflow6Simu
     return simulation
 
 
+def make_recharge(idomain: xr.DataArray) -> mf6.Recharge:
+    idomain_l1 = idomain.sel(layer=1)
+    recharge = xr.zeros_like(idomain_l1, dtype=float)
+    recharge[:, 0] = np.nan
```

Comment:
are you assigning nan to all x for y =0? (if so, why is that? if something else, what exactly?)

Reply:
The opposite, all cells in the first column (x) are assigned ``np.nan``. This is consistent with the MetaSWAP model, where these cells are inactive, and thus no mapping should take place to these cells.

Reply:
I added a comment to clarify